sources/packages/backend/apps/db-migrations/src/sql/SFASPartTimeApplications/Add-col-py-maximum-support.sql

```diff
@@ -0,0 +1,39 @@
+-- Add column cspt_award for SFAS Part Time Application.
+ALTER TABLE 
+  sims.sfas_part_time_applications 
+ADD 
+  COLUMN IF NOT EXISTS cspt_award NUMERIC(10);
```

Comment:
Not completely sure but I think we had an agreement in the past to not use `IF EXISTS` and `IF NOT EXISTS` anymore. Other Devs can confirm.

Reply:
@sh16011993 is right. We have an agreement not to use them as those ones can potentially bypass/ignore the errors which needs to be reported on migration

Reply:
Sounds good to me. I'll remove `IF NOT EXISTS`.